tests/common.py

```diff
@@ -1129,6 +1134,16 @@ async def get_system_health_info(hass, domain):
     return await hass.data["system_health"][domain].info_callback(hass)
 
 
+async def get_integration_updates(
+    hass: HomeAssistant,
+    domain: str,
+) -> list[UpdateDescription]:
+    """Get updates for an integration."""
+    await async_setup_component(hass, UPDATE_DOMAIN, {})
+    update_data: UpdateData = hass.data[UPDATE_DOMAIN]
```

Comment:
I think we should make a helper similar to the diagnostics integration instead, which doesn't access hass.data directly in the tests:
https://github.com/home-assistant/core/blob/64277058b5ba6fb10029553422695964204f0ebb/tests/components/diagnostics/__init__.py#L19-L22

Reply:
I will exclude this and the scaffold logic from this PR.